docs/tutorials/config.md

```diff
@@ -46,6 +46,7 @@ player.getConfiguration();
        defaultBandwidthEstimate: 500000
        enabled: true
        restrictions: Object
+       safeSeekOffset: 5
```

Comment:
I think this does not belong under "abr".  You seem to have another copy under "streaming".

Reply:
my bad